python/GafferImageTest/OpenImageIOReaderTest.py

```diff
@@ -288,6 +288,56 @@ def testAvailableFrames( self ) :
 		reader["fileName"].setValue( self.missingFileName )
 		self.assertEqual( reader["availableFrames"].getValue(), IECore.IntVectorData( [] ) )
 
+	def testFileValid( self ) :
+
+		testFile = self.temporaryDirectory() / "single_file.exr"
+
+		reader = GafferImage.OpenImageIOReader()
+		reader["fileName"].setValue( pathlib.Path( testFile ) )
```

Comment:
I think `testFile` is already a `pathlib.Path()` isn't it?

Reply:
fixed